tools/generic/MASTG-TOOL-0042.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Please use: `MASTG-TOOL-0104`

Since today you can go to the tools overview and sort by ID to find out which one would be the next to use: 

https://mas.owasp.org/MASTG/tools/


Thanks!

Reply:
Thanks, will do